src/classes/STG_InstallScript_TEST.cls

```diff
@@ -217,9 +214,7 @@ public with sharing class STG_InstallScript_TEST {
         npe04Settings.npe4__DISABLE_RelationshipCM_trigger__c = flagValue;
         npe04Settings.npe4__DISABLE_RelationshipContacts_trigger__c = flagValue;
         npe04Settings.npe4__DISABLE_Relationships_trigger__c = flagValue;
-        settingsToUpdate.add(npe04Settings);
-        
-        update settingsToUpdate;
+        settingsToUpdate.add(npe04Settings);   
```

Comment:
why did you leave in this call to settingsToUpdate.add(), but removed it from other places in this test class?